### PR TITLE
bug fix to na_ontap_interface

### PR DESCRIPTION
**Pull request by carchi8py**
##### SUMMARY
The protocols for ontap interface can be fc-nvme or fcp, only fc-nvme was listed. 

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
-  na_ontap_interface.py

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/storage/netapp/na_ontap_interface.py [hunk 1]
```diff
@@ -156,26 +156,27 @@ class NetAppOntapInterface(object):
...
-            home_port=dict(type='str'),
-            role=dict(type='str'),
-            address=dict(type='str'),
-            netmask=dict(type='str'),
-            vserver=dict(type='str', required=True),
-            firewall_policy=dict(type='str'),
-            failover_policy=dict(type='str'),
-            admin_status=dict(type='str', choices=['up', 'down']),
-            subnet_name=dict(type='str'),
-            is_auto_revert=dict(type='bool'),
```
Comment (lonico) [inline lib/ansible/modules/storage/netapp/na_ontap_interface.py]: should be 'bool'

File: lib/ansible/modules/storage/netapp/na_ontap_interface.py [hunk 2]
```diff
@@ -156,26 +156,27 @@ class NetAppOntapInterface(object):
...
+            home_node=dict(required=False, type='str', default=None),
+            home_port=dict(required=False, type='str'),
+            role=dict(required=False, type='str'),
+            address=dict(required=False, type='str'),
+            netmask=dict(required=False, type='str'),
+            vserver=dict(required=True, type='str'),
+            firewall_policy=dict(required=False, type='str', default=None),
+            failover_policy=dict(required=False, type='str', default=None),
+            admin_status=dict(required=False, choices=['up', 'down']),
+            subnet_name=dict(required=False, type='str'),
```
Comment (lonico) [inline lib/ansible/modules/storage/netapp/na_ontap_interface.py]: should be 'bool'

File: lib/ansible/modules/storage/netapp/na_ontap_interface.py [hunk 3]
```diff
@@ -156,26 +156,27 @@ class NetAppOntapInterface(object):
...
+            is_auto_revert=dict(required=False, type=bool, default=None),
```
Comment (lonico) [inline lib/ansible/modules/storage/netapp/na_ontap_interface.py]: should be 'bool'

Comment (thedoubl3j) [reply lib/ansible/modules/storage/netapp/na_ontap_interface.py]: @carchi8py @lonico can we get this committed so we can get this in?

Comment (thedoubl3j) [reply lib/ansible/modules/storage/netapp/na_ontap_interface.py]: I don't have write access to the repo or I would have opened a PR ;)

Comment (gundalow) [reply lib/ansible/modules/storage/netapp/na_ontap_interface.py]:
```suggestion
            is_auto_revert=dict(required=False, type='bool', default=None),
```

**Review by thedoubl3j — APPROVED** [review]

**Pull request merged**